content/docs/command-reference/exp/init.md

```diff
@@ -17,37 +18,52 @@ usage: dvc exp init [-h] [-q | -v] [--run] [--interactive] [-f]
 ## Description
 
 `dvc exp init` helps you quickly get started with experiments. It reduces
-boilerplate for initializing [pipeline](/doc/command-reference/dag) stages in a
-`dvc.yaml` file by assuming defaults about the location of your data,
-[parameters](/doc/command-reference/params), source code, models,
-[metrics](/doc/command-reference/metrics) and
-[plots](/doc/command-reference/plots), which can be customized through config.
+boilerplate DVC procedures by creating a `dvc.yaml` file that assumes default
+location of your input data, <abbr>parameters</abbr>, source code, models,
+<abbr>metrics</abbr> and [plots](/doc/command-reference/plots). These locations
+can be customized through the options of this command or via config files.
+Standard repository structure:
 
-It also offers guided `--interactive` mode for creating a stage to be
-[`exp run`](/doc/command-reference/exp/run) later. `dvc exp init` supports
-creating different types of stages, eg: `dl` if you are doing deep learning,
-which uses [dvclive](/doc/dvclive) to monitor and checkpoint progress during
-training of machine learning models.
+```
+├── data/
+├── metrics.json
+├── models/
+├── params.yaml  # required
+├── plots/
+└── src/
+```
+
+> Note that `params.yaml` is the only required file (see `dvc params`).
+
+To use this feature, provide a `command` argument or use the `--interactive`
```

Comment:
`this feature` refers to?

Reply:
"this feature" referred to `exp init`. I didn't want to say "this command" because `exp init` accepts a `command` argument so it's confusing (same happens in `run` and `stage add`). Anyway, I'm just going to be explicit every time instead. Solved